service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/InputBundle.java

```diff
@@ -18,6 +18,7 @@
 import java.util.stream.Collectors;
 
 import static uk.nhs.adaptors.gp2gp.ehr.utils.ExtensionMappingUtils.filterExtensionByUrl;
+import static uk.nhs.adaptors.gp2gp.ehr.utils.ResourceExtractor.*;
```

Comment:
don't use wildcard imports, specify the method you are pulling in

Reply:
Got rid of that on rolling back, but will keep that in mind, thanks